app/src/main/java/com/android/sample/ui/createRecipe/IconDropdownMenu.kt

```diff
@@ -41,7 +43,8 @@ fun IconDropdownMenu(
     modifier: Modifier = Modifier
 ) {
   val isDropDownExpanded = remember { mutableStateOf(false) }
-  val iconOptions = listOf(IconType.Fire, IconType.Salt, IconType.Mortar, IconType.Axe)
+
+  val iconOptions = listOf(IconType("fire"), IconType("salt"), IconType("mortar"), IconType("axe"))
```

Comment:
Great work on this implementation! Just a small detail: could you please move these strings to `C.kt` as constants? It will help maintain consistency and reusability across the project. Thanks!